cmd/stork/stork.go

```diff
@@ -209,6 +209,10 @@ func main() {
 		},
 	}
 
+	// Export USE_GKE_GCLOUD_AUTH_PLUGIN=True
+	log.Warnf("Export USE_GKE_GCLOUD_AUTH_PLUGIN=True")
+	os.Setenv("USE_GKE_GCLOUD_AUTH_PLUGIN", "True")
```

Comment:
Does it require "True" or "true"

Reply:
It is "True" https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke

Reply:
is setting same env var also required in integration test container?

Reply:
good point. I actually didn't face any issues when I didn't set this env variable. Will keep this in mind when we run integration tests on GKE. If we are facing issues in fetching the kubeconfig, we can set it then.